server/pbench/bin/pbench-sync-package-tarballs

```diff
@@ -91,16 +93,12 @@ if [ -s $tmp/mail.log ]; then
     nerrs=$(wc -l < $tmp/mail.log)
 fi
 
-# if there are errors, send mail
-if [ "$nerrs" -gt 0 ]  ;then
-    index_content=$tmp/index_mail_contents
-    cat << EOF > $index_content
+index_content=$tmp/index_mail_contents
+cat << EOF > $index_content
 $PROG: Processed $nprocessed result tar balls, with $nerrs errors
 
 EOF
-    cat $tmp/mail.log >> $index_content
-    pbench-report-status --name $PROG --timestamp $TS --type status $index_content
-fi
+cat $tmp/mail.log >> $index_content
+pbench-report-status --name $PROG --timestamp $TS --type status $index_content
```

Comment:
Should we be doing this in this command?  Seems like `pbench-sync-satellite` will report on this instead.

Reply:
Agree. `pbench-sync-satellite` will report about the results.